frontend/pages/index.tsx

```diff
@@ -23,21 +23,20 @@ export const getServerSideProps = async (
     headers: { cookie: context.req.headers.cookie },
   })
 
-  if (res.ok) {
-    return {
-      redirect: {
-        destination: DASHBOARD_ROUTE,
-        permanent: false,
-      },
-    }
-  } else {
-    return {
-      redirect: {
-        destination: `/api/user/clear-cookies/?next=${DASHBOARD_ROUTE}`,
-        permanent: false,
-      },
-    }
+  if (
+    res.status === 403 &&
```

Comment:
idk Javascript but is there a reason you don't need to await for status but do need to for the json?

Reply:
no clue tbh, i've just accepted it ig lmao.